tests/onboarding/test_onboarding_import_seed.py

```diff
@@ -23,11 +23,15 @@ def keys_screen(main_window) -> KeysView:
 
 @allure.testcase('https://ethstatus.testrail.net/index.php?/cases/view/703040', 'Import: 12 word seed phrase')
 @pytest.mark.case(703040)
-@pytest.mark.parametrize('user_account', [constants.user.user_account_two])
-def test_import_seed_phrase(aut: AUT, keys_screen, main_window, user_account):
+@pytest.mark.parametrize('user_account', [constants.user.user_account_one])
+@pytest.mark.parametrize('type_of_selection', [
+    pytest.param('auto'),
+    pytest.param('enter')
```

Comment:
please change string values that you are using as boolean to boolean
like 'auto' and 'enter'

Reply:
Please, check, not sure if I got what you mean, but I changed to autocomplete with params True and False